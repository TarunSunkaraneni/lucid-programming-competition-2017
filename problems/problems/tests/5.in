8 1500
stuck 4
i18n 3
polygons 5
elavator 4
polygons 5
polygons 2
kdtree 3
eating_pizza 6
problems 7
resize 7
i18n 3
polygons 6
polygons 7
problems 5
problems 6
eating_pizza 7
elavator 7
resize 2
stuck 0
problems 6
resize 7
resize 5
problems 2
eating_pizza 0
i18n 6
polygons 7
problems 1
kdtree 6
elavator 2
i18n 5
stuck 4
eating_pizza 3
stuck 7
stuck 2
problems 1
stuck 0
polygons 3
polygons 7
stuck 2
eating_pizza 0
stuck 2
problems 0
eating_pizza 2
kdtree 4
eating_pizza 3
eating_pizza 2
resize 0
i18n 7
elavator 0
i18n 3
resize 2
stuck 5
elavator 3
kdtree 0
problems 3
stuck 4
resize 5
stuck 4
kdtree 5
problems 0
stuck 7
elavator 2
i18n 7
stuck 3
elavator 1
elavator 4
kdtree 6
problems 6
eating_pizza 5
elavator 0
eating_pizza 0
problems 1
stuck 1
i18n 0
stuck 4
problems 1
kdtree 0
kdtree 2
stuck 7
i18n 7
i18n 6
eating_pizza 1
stuck 4
kdtree 6
elavator 0
kdtree 2
stuck 4
polygons 1
resize 5
problems 6
stuck 4
kdtree 1
elavator 1
resize 1
stuck 6
polygons 6
problems 2
stuck 4
i18n 2
resize 5
polygons 3
eating_pizza 1
problems 4
elavator 7
resize 0
stuck 6
eating_pizza 2
eating_pizza 6
eating_pizza 6
stuck 5
eating_pizza 1
elavator 4
kdtree 0
problems 5
problems 0
polygons 5
problems 4
polygons 2
kdtree 5
eating_pizza 7
stuck 6
eating_pizza 4
problems 7
stuck 7
stuck 4
problems 1
eating_pizza 3
stuck 4
eating_pizza 1
i18n 6
problems 2
eating_pizza 6
resize 3
stuck 4
problems 3
polygons 6
polygons 6
i18n 3
i18n 5
elavator 5
kdtree 2
stuck 7
eating_pizza 4
problems 7
i18n 0
i18n 3
kdtree 2
resize 2
resize 7
eating_pizza 1
resize 6
kdtree 3
problems 3
polygons 6
polygons 0
polygons 4
elavator 7
polygons 5
eating_pizza 2
stuck 6
eating_pizza 4
kdtree 1
elavator 1
i18n 0
elavator 7
resize 6
polygons 7
kdtree 2
problems 6
i18n 5
stuck 5
stuck 7
polygons 1
elavator 7
eating_pizza 2
polygons 1
polygons 0
elavator 0
i18n 2
elavator 6
eating_pizza 4
problems 4
polygons 0
problems 4
resize 7
kdtree 0
elavator 2
eating_pizza 3
elavator 7
problems 0
i18n 2
i18n 7
elavator 6
problems 5
problems 0
resize 4
resize 7
problems 6
polygons 0
kdtree 0
polygons 1
kdtree 6
elavator 6
polygons 6
elavator 7
i18n 6
eating_pizza 4
problems 6
elavator 2
i18n 3
polygons 0
i18n 3
i18n 2
polygons 3
stuck 7
i18n 0
kdtree 3
i18n 1
resize 1
elavator 5
eating_pizza 7
resize 3
i18n 3
elavator 3
resize 5
i18n 3
kdtree 3
i18n 2
elavator 4
eating_pizza 7
polygons 0
i18n 1
resize 4
polygons 6
stuck 4
eating_pizza 6
problems 0
resize 4
stuck 4
eating_pizza 6
elavator 0
stuck 6
stuck 3
i18n 4
problems 6
stuck 7
resize 3
resize 1
polygons 5
stuck 6
resize 6
stuck 3
problems 6
eating_pizza 2
i18n 6
eating_pizza 6
elavator 0
eating_pizza 4
polygons 7
polygons 6
kdtree 2
polygons 5
kdtree 0
stuck 2
kdtree 3
i18n 7
elavator 0
polygons 4
i18n 7
polygons 5
problems 7
kdtree 3
elavator 7
elavator 2
resize 4
kdtree 3
resize 6
resize 1
polygons 4
polygons 5
elavator 2
i18n 7
polygons 6
eating_pizza 5
resize 2
problems 6
kdtree 3
eating_pizza 7
resize 6
elavator 2
i18n 1
polygons 5
elavator 5
problems 1
resize 2
kdtree 3
stuck 0
stuck 6
i18n 4
polygons 1
i18n 6
elavator 2
polygons 3
polygons 0
stuck 0
problems 0
stuck 6
i18n 1
polygons 3
problems 5
kdtree 3
stuck 3
stuck 6
polygons 1
polygons 1
stuck 2
polygons 6
stuck 4
polygons 0
kdtree 5
kdtree 3
resize 7
problems 4
stuck 5
stuck 1
eating_pizza 3
eating_pizza 5
i18n 2
resize 4
elavator 6
stuck 2
kdtree 1
polygons 5
i18n 1
resize 1
elavator 4
polygons 2
stuck 5
elavator 3
stuck 4
elavator 7
kdtree 1
problems 0
problems 4
eating_pizza 5
elavator 1
resize 5
problems 0
resize 2
problems 7
kdtree 4
resize 1
problems 4
problems 5
problems 6
stuck 0
problems 6
i18n 4
i18n 6
resize 2
i18n 4
stuck 1
polygons 7
stuck 0
polygons 6
problems 0
polygons 5
stuck 4
kdtree 3
resize 7
problems 1
i18n 3
eating_pizza 0
resize 5
i18n 3
polygons 1
eating_pizza 7
i18n 6
polygons 3
stuck 1
stuck 0
eating_pizza 7
stuck 5
problems 5
elavator 4
resize 4
elavator 2
stuck 4
resize 2
i18n 4
stuck 5
eating_pizza 3
polygons 7
resize 5
eating_pizza 0
i18n 5
polygons 1
elavator 0
stuck 7
eating_pizza 5
polygons 3
eating_pizza 5
problems 0
problems 7
eating_pizza 3
kdtree 6
resize 5
eating_pizza 1
problems 0
kdtree 4
resize 3
problems 7
elavator 7
elavator 4
i18n 5
polygons 1
i18n 0
stuck 4
kdtree 7
stuck 0
polygons 5
elavator 2
kdtree 1
i18n 1
kdtree 6
polygons 1
stuck 5
kdtree 1
elavator 0
elavator 6
i18n 2
resize 6
polygons 7
i18n 3
problems 1
problems 1
i18n 0
polygons 1
kdtree 3
eating_pizza 0
eating_pizza 7
polygons 4
resize 2
problems 0
i18n 2
polygons 4
eating_pizza 7
elavator 0
elavator 1
stuck 5
polygons 5
resize 6
i18n 6
problems 0
polygons 1
polygons 4
polygons 5
stuck 5
kdtree 1
kdtree 0
i18n 5
polygons 5
i18n 3
problems 0
problems 7
resize 4
i18n 3
stuck 7
resize 2
problems 0
resize 7
eating_pizza 1
eating_pizza 1
problems 7
eating_pizza 0
kdtree 6
i18n 1
i18n 5
resize 2
elavator 5
polygons 4
problems 1
eating_pizza 1
stuck 6
stuck 5
resize 7
kdtree 0
kdtree 6
kdtree 7
kdtree 1
resize 3
polygons 3
eating_pizza 5
polygons 0
elavator 4
kdtree 5
polygons 2
resize 7
resize 0
polygons 3
problems 6
eating_pizza 4
polygons 5
kdtree 7
polygons 0
problems 4
i18n 2
elavator 1
resize 3
stuck 4
kdtree 7
problems 0
problems 6
resize 5
eating_pizza 6
polygons 4
kdtree 0
kdtree 5
polygons 1
eating_pizza 6
eating_pizza 1
problems 7
resize 2
elavator 4
elavator 5
kdtree 0
eating_pizza 7
polygons 7
i18n 0
elavator 6
elavator 7
stuck 2
stuck 5
resize 6
elavator 1
problems 3
problems 5
problems 3
problems 1
elavator 4
kdtree 6
polygons 5
problems 7
stuck 6
polygons 1
eating_pizza 4
i18n 7
stuck 4
stuck 6
i18n 0
stuck 2
kdtree 1
kdtree 3
polygons 6
eating_pizza 6
problems 5
resize 6
elavator 1
i18n 5
problems 3
elavator 1
kdtree 0
stuck 3
stuck 0
eating_pizza 3
eating_pizza 1
polygons 6
eating_pizza 3
elavator 3
kdtree 6
stuck 2
problems 6
i18n 3
resize 7
kdtree 2
elavator 0
elavator 1
problems 1
stuck 0
stuck 3
problems 4
stuck 5
i18n 6
i18n 5
kdtree 4
elavator 6
elavator 4
stuck 6
resize 6
polygons 2
eating_pizza 3
kdtree 1
stuck 1
polygons 7
polygons 3
eating_pizza 4
i18n 4
resize 6
i18n 3
eating_pizza 1
polygons 4
kdtree 0
elavator 2
i18n 6
polygons 3
resize 3
i18n 0
elavator 1
polygons 3
polygons 3
problems 2
problems 4
eating_pizza 1
resize 1
problems 5
i18n 6
problems 5
elavator 2
i18n 7
eating_pizza 4
problems 7
i18n 5
problems 6
resize 5
problems 3
resize 4
kdtree 5
eating_pizza 0
i18n 1
eating_pizza 0
problems 6
problems 4
resize 5
kdtree 2
eating_pizza 3
problems 1
eating_pizza 5
eating_pizza 4
polygons 7
problems 4
kdtree 7
i18n 0
polygons 7
problems 7
problems 2
stuck 2
problems 2
polygons 5
stuck 5
kdtree 1
polygons 3
stuck 2
i18n 1
polygons 5
polygons 4
eating_pizza 1
polygons 2
eating_pizza 6
problems 4
elavator 1
resize 0
i18n 5
elavator 5
problems 3
polygons 7
i18n 3
stuck 2
resize 0
elavator 6
stuck 2
problems 7
problems 1
kdtree 2
resize 0
stuck 3
resize 5
polygons 3
resize 7
kdtree 0
problems 6
polygons 5
eating_pizza 5
problems 1
i18n 2
kdtree 1
resize 0
stuck 1
stuck 3
eating_pizza 3
i18n 5
problems 0
eating_pizza 6
elavator 2
elavator 6
eating_pizza 7
eating_pizza 1
resize 4
polygons 5
stuck 2
elavator 6
kdtree 1
stuck 5
i18n 2
elavator 6
resize 3
problems 0
elavator 2
polygons 5
polygons 0
elavator 6
elavator 0
problems 7
eating_pizza 2
elavator 1
stuck 5
eating_pizza 3
stuck 2
i18n 3
problems 5
kdtree 5
polygons 4
elavator 2
polygons 3
polygons 7
resize 1
kdtree 0
resize 5
kdtree 4
i18n 5
elavator 3
polygons 3
problems 4
kdtree 0
elavator 6
stuck 5
i18n 5
problems 0
i18n 6
kdtree 5
i18n 6
polygons 1
eating_pizza 2
polygons 6
i18n 7
resize 6
polygons 5
eating_pizza 1
i18n 1
stuck 1
i18n 2
i18n 6
problems 0
i18n 7
i18n 3
elavator 4
eating_pizza 3
i18n 1
kdtree 2
kdtree 5
kdtree 2
resize 1
polygons 7
elavator 7
elavator 5
stuck 6
eating_pizza 1
problems 1
polygons 4
stuck 5
eating_pizza 0
eating_pizza 7
elavator 3
elavator 5
elavator 3
eating_pizza 7
stuck 2
i18n 1
elavator 5
elavator 2
elavator 3
i18n 3
eating_pizza 6
stuck 4
stuck 5
elavator 2
kdtree 6
eating_pizza 5
resize 5
eating_pizza 6
problems 4
polygons 0
eating_pizza 7
eating_pizza 1
eating_pizza 3
resize 3
problems 3
stuck 0
stuck 1
eating_pizza 7
kdtree 7
kdtree 7
problems 4
stuck 0
polygons 0
polygons 6
resize 6
elavator 3
elavator 3
resize 3
elavator 7
problems 0
polygons 2
kdtree 6
eating_pizza 6
resize 6
elavator 5
stuck 3
stuck 1
stuck 1
problems 4
problems 3
problems 0
eating_pizza 0
i18n 7
polygons 1
kdtree 2
polygons 7
problems 2
problems 5
eating_pizza 3
problems 5
eating_pizza 0
polygons 5
problems 3
kdtree 2
elavator 1
eating_pizza 7
eating_pizza 1
kdtree 7
eating_pizza 2
eating_pizza 4
resize 2
kdtree 5
problems 1
i18n 4
kdtree 4
problems 1
kdtree 2
eating_pizza 6
problems 7
kdtree 7
polygons 4
eating_pizza 4
problems 0
problems 7
problems 5
resize 0
problems 4
resize 3
i18n 4
stuck 4
resize 4
polygons 2
polygons 7
polygons 4
polygons 0
polygons 7
stuck 7
elavator 6
problems 2
elavator 1
i18n 1
kdtree 5
kdtree 0
stuck 3
stuck 2
resize 5
problems 4
kdtree 6
i18n 3
stuck 4
i18n 0
resize 2
elavator 7
elavator 6
stuck 5
eating_pizza 4
eating_pizza 0
stuck 1
kdtree 6
stuck 1
problems 7
problems 7
elavator 7
resize 1
elavator 7
eating_pizza 1
resize 7
eating_pizza 2
eating_pizza 4
stuck 2
polygons 0
problems 5
elavator 3
resize 4
elavator 1
i18n 5
resize 3
stuck 7
i18n 3
polygons 6
stuck 3
resize 0
eating_pizza 3
problems 7
problems 4
elavator 7
kdtree 5
problems 4
kdtree 0
kdtree 1
resize 4
eating_pizza 6
elavator 4
eating_pizza 1
problems 7
elavator 2
resize 4
kdtree 2
kdtree 4
stuck 5
eating_pizza 7
eating_pizza 1
resize 0
resize 2
stuck 6
resize 6
eating_pizza 4
eating_pizza 4
eating_pizza 4
kdtree 0
problems 2
elavator 5
eating_pizza 6
kdtree 0
kdtree 7
eating_pizza 3
kdtree 7
stuck 5
problems 7
stuck 1
i18n 0
stuck 6
resize 4
eating_pizza 5
stuck 7
problems 2
kdtree 4
polygons 1
polygons 2
resize 7
problems 4
kdtree 1
i18n 2
stuck 5
problems 4
polygons 5
polygons 4
i18n 3
problems 5
polygons 0
polygons 7
elavator 7
elavator 3
elavator 7
kdtree 4
problems 6
problems 3
elavator 0
kdtree 7
polygons 0
eating_pizza 6
kdtree 1
stuck 3
resize 1
i18n 6
i18n 0
elavator 6
problems 5
polygons 0
elavator 5
kdtree 7
problems 1
resize 5
i18n 0
i18n 4
resize 7
polygons 6
resize 7
elavator 1
i18n 2
resize 3
i18n 3
stuck 3
eating_pizza 0
eating_pizza 7
i18n 3
eating_pizza 0
elavator 0
resize 0
resize 0
kdtree 7
resize 1
resize 3
elavator 1
problems 4
elavator 6
stuck 1
polygons 4
kdtree 7
kdtree 5
stuck 4
problems 0
resize 2
eating_pizza 5
kdtree 2
i18n 0
elavator 6
i18n 2
i18n 6
kdtree 3
problems 1
stuck 6
polygons 2
resize 0
stuck 5
eating_pizza 7
elavator 7
problems 3
polygons 3
problems 6
kdtree 4
resize 3
problems 1
kdtree 2
resize 3
polygons 2
eating_pizza 0
elavator 4
polygons 0
stuck 7
resize 7
i18n 7
eating_pizza 1
eating_pizza 5
resize 2
kdtree 0
polygons 2
eating_pizza 5
i18n 4
elavator 7
problems 6
resize 7
polygons 3
i18n 6
stuck 0
i18n 5
problems 7
i18n 0
stuck 3
resize 6
stuck 0
polygons 4
i18n 5
problems 4
problems 0
problems 2
eating_pizza 7
i18n 6
problems 6
eating_pizza 4
kdtree 3
polygons 3
kdtree 6
kdtree 2
problems 4
problems 5
resize 2
i18n 6
stuck 7
resize 4
elavator 0
polygons 4
i18n 0
stuck 1
resize 6
problems 1
kdtree 6
polygons 0
resize 3
eating_pizza 6
kdtree 0
eating_pizza 5
kdtree 4
i18n 0
elavator 6
eating_pizza 7
kdtree 3
resize 3
kdtree 0
problems 0
resize 4
problems 3
stuck 3
problems 0
elavator 2
kdtree 0
kdtree 5
eating_pizza 1
problems 4
resize 3
problems 0
kdtree 7
resize 4
kdtree 4
kdtree 5
i18n 7
eating_pizza 4
resize 2
polygons 0
polygons 4
stuck 1
kdtree 2
resize 3
i18n 0
polygons 7
stuck 4
eating_pizza 0
eating_pizza 7
resize 6
elavator 3
elavator 2
elavator 6
problems 5
polygons 2
polygons 2
polygons 1
kdtree 7
elavator 2
resize 6
i18n 5
resize 0
resize 1
polygons 1
i18n 7
elavator 3
kdtree 2
stuck 7
i18n 2
elavator 4
eating_pizza 4
stuck 6
i18n 2
stuck 2
eating_pizza 0
resize 6
resize 0
kdtree 5
i18n 4
kdtree 0
elavator 1
polygons 0
resize 0
stuck 3
eating_pizza 1
elavator 6
problems 7
eating_pizza 6
elavator 1
eating_pizza 0
i18n 3
stuck 6
i18n 5
resize 4
polygons 7
resize 1
eating_pizza 5
stuck 7
eating_pizza 0
i18n 6
elavator 4
kdtree 1
elavator 4
i18n 3
resize 0
eating_pizza 3
eating_pizza 4
elavator 5
eating_pizza 0
stuck 6
polygons 0
stuck 1
problems 2
resize 3
i18n 6
problems 2
resize 5
polygons 2
eating_pizza 7
stuck 0
eating_pizza 3
resize 0
resize 7
problems 2
polygons 4
elavator 6
polygons 2
polygons 3
problems 7
eating_pizza 6
kdtree 4
resize 6
elavator 2
kdtree 4
kdtree 2
eating_pizza 0
eating_pizza 2
kdtree 2
polygons 7
i18n 6
stuck 4
polygons 3
polygons 6
elavator 2
i18n 4
polygons 0
polygons 0
eating_pizza 0
stuck 7
stuck 0
resize 5
eating_pizza 4
stuck 6
i18n 0
resize 3
stuck 1
problems 2
i18n 5
eating_pizza 0
stuck 7
eating_pizza 4
elavator 0
kdtree 4
polygons 4
eating_pizza 3
kdtree 1
kdtree 1
resize 2
eating_pizza 5
resize 1
resize 4
elavator 5
polygons 4
i18n 2
i18n 5
eating_pizza 1
elavator 3
problems 6
stuck 0
polygons 5
polygons 5
problems 5
elavator 1
eating_pizza 3
stuck 7
kdtree 2
problems 1
problems 0
problems 2
i18n 1
elavator 1
elavator 2
stuck 4
stuck 7
eating_pizza 4
stuck 4
elavator 2
elavator 3
polygons 2
stuck 5
resize 3
eating_pizza 1
kdtree 5
problems 5
problems 2
stuck 5
problems 0
resize 0
stuck 5
problems 2
kdtree 6
eating_pizza 3
problems 4
resize 3
elavator 2
stuck 7
resize 3
i18n 1
kdtree 6
polygons 2
kdtree 0
elavator 6
elavator 0
elavator 1
stuck 4
problems 2
problems 1
elavator 6
elavator 6
eating_pizza 7
problems 4
eating_pizza 3
i18n 3
resize 2
kdtree 0
elavator 4
i18n 3
polygons 2
polygons 4
i18n 4
kdtree 2
polygons 5
elavator 4
kdtree 3
elavator 2
stuck 7
problems 1
eating_pizza 6
elavator 3
i18n 1
eating_pizza 1
problems 1
i18n 3
kdtree 0
i18n 2
polygons 7
stuck 5
stuck 0
resize 2
kdtree 6
kdtree 4
stuck 7
stuck 3
problems 7
eating_pizza 2
stuck 5
stuck 5
i18n 5
eating_pizza 4
stuck 6
polygons 7
stuck 4
eating_pizza 7
i18n 2
elavator 7
eating_pizza 2
polygons 3
stuck 7
resize 3
problems 7
eating_pizza 5
problems 4
polygons 3
eating_pizza 5
resize 1
problems 0
i18n 6
polygons 1
stuck 2
eating_pizza 4
stuck 3
stuck 4
eating_pizza 1
resize 6
eating_pizza 1
kdtree 7
eating_pizza 7
stuck 7
kdtree 1
polygons 5
elavator 0
elavator 7
polygons 7
eating_pizza 2
resize 4
elavator 7
problems 1
eating_pizza 5
resize 3
problems 1
i18n 6
i18n 0
stuck 2
stuck 6
resize 1
eating_pizza 5
kdtree 5
problems 5
eating_pizza 5
i18n 5
elavator 2
resize 2
elavator 7
eating_pizza 4
polygons 3
problems 4
polygons 4
resize 1
eating_pizza 2
resize 1
stuck 2
elavator 6
stuck 1
resize 5
eating_pizza 6
resize 6
eating_pizza 6
problems 4
elavator 0
polygons 0
eating_pizza 0
eating_pizza 3
polygons 4
problems 2
elavator 6
polygons 0
i18n 0
stuck 4
eating_pizza 3
resize 3
resize 7
problems 0
elavator 7
i18n 5
i18n 5
polygons 0
polygons 1
i18n 6
i18n 4
elavator 5
kdtree 2
elavator 5
resize 7
problems 1
resize 4
problems 5
elavator 3
kdtree 3
i18n 0
i18n 0
eating_pizza 4
resize 7
stuck 2
elavator 1
eating_pizza 6
kdtree 7
kdtree 4
problems 6
problems 1
i18n 2
problems 0
elavator 2
resize 7
kdtree 1
kdtree 1
stuck 0
polygons 7
elavator 5
elavator 5
problems 0
problems 5
elavator 5
i18n 7
stuck 1
eating_pizza 1
problems 0
polygons 2
problems 6
kdtree 1
polygons 7
i18n 0
resize 1
problems 4
elavator 6
eating_pizza 7
elavator 0
resize 0
eating_pizza 1
kdtree 0
i18n 1
