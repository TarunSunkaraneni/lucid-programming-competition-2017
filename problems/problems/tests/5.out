3657632646751708
